main.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.start_console
+end
+
+def list_of_options
+  puts
+  puts 'Please choose an option by entering a number:'
+  puts '1 - list all books'
+  puts '2 - list all people'
+  puts '3 - create a person'
+  puts '4 - create a book'
+  puts '5 - create a renter'
```

Comment:
Good job fixing the bug error experienced when creating a student, while that is fixed, a similar error occurs when a rental is attempted to be created. Consider resolving this in order for the app to behave correctly

<img width="696" alt="hfyfyufu" src="https://github.com/fredo509/OOP-school-library/assets/84930355/64ce1a20-7383-4587-86bd-22774f9fd5f7">


[OPTIONAL} I am marking this as optional since my colleague didn't notice it, consider checking the app when a teacher is being created, it throws some error that crashes the app.